README.md

```diff
@@ -251,32 +251,28 @@ not set, it will mount "/pool" as "C:/pool".
 
 # Installing a binary release
 
-Latest binary files are available on GitHub:
+Latest binary files are available at [GitHub releases](https://github.com/openzfsonwindows/ZFSin/releases)
 
-https://github.com/openzfsonwindows/ZFSin/releases
 
-Download your preferred package. Either debug, or release.
+If you are running windows 10 with secure boot on and/or installing an older release you will need to enable unsigned drivers from an elevated CMD:
 
-Enable unsigned drivers:
+ 
+* `bcdedit.exe -set testsigning on `
+* Then **reboot**. After restart it should have _Test Mode_ bottom right corner of the screen.
 
-* bcdedit.exe -set testsigning on
-
-Then **reboot**. After restart it should have _Test Mode_ bottom right
-corner of the screen.
 
 After that either 
 
-* Run OpenZFSOnWindows.exe installer to install.
-* Windows should popup the "Windows can't verify the publisher of this driver software"
-* Click "Install this driver software anyway"
+* Run OpenZFSOnWindows.exe installer to install
+* *Would you like to install device software?* should pop up, click install
+  * If installing an unsigned release, click "Install anyway" in the "unknown developer" popup
```

Comment:
Running the installer, when I produce it, should not have "unknown developer" warning any more?

Reply:
Correct, the signed releases should be good to go without test mode.

A fresh install of Windows 10 1607 or higher with secure boot enabled, is the only exception, since that requires drivers to be signed with a EV cert to install them.